Node JS/session fswd 01/UI/food-adda/src/pages/Login.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from "react";
+import { Form, Button } from "react-bootstrap";
+import { Link } from "react-router-dom";
+import { toast } from "react-toastify";
+import { Signup } from "../components/Signup";
+
+export const Login = () => {
+  // State variables
+  const [modalShow, setModalShow] = useState(false);
```

Comment:
Its better to have a seperate signup page instead of having it in a modal.
1. It would be good for deep links
2. Good for SEO